src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.Before;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService = new RegistrationServiceImpl();
+
+    @Before
+    void before() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void register_nullAge_notOk() {
+        User user = new User(1L,"Bob","12345",0);
+        assertThrows(RegistrationException.class, () -> registrationService.register(user));
+    }
+
+    @Test
+    void register_nullLogin_notOk() {
+        User user = new User(1L,null,"1234567",20);

```

Comment:
create constants like VALID_PASSWORD and reuse them where you need them, or setup default valid user in `init`, and just set his login to null in this case